From: Benjamin Kaduk <kaduk@mit.edu>
Date: Sat, 6 Sep 2014 01:29:29 -0400
Subject: fix memset call

---
 tests/rx/event-t.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/rx/event-t.c b/tests/rx/event-t.c
index 3e7ab0a..2812fa5 100644
--- a/tests/rx/event-t.c
+++ b/tests/rx/event-t.c
@@ -105,7 +105,7 @@ main(void)
     pthread_mutex_init(&eventMutex, NULL);
     pthread_cond_init(&eventCond, NULL);
 
-    memset(events, sizeof(events), 0);
+    memset(events, 0, sizeof(events));
     pthread_mutex_init(&eventListMutex, NULL);
 
     /* Start up the event system */
